Content.Client/RoundEnd/RoundEndSummaryWindow.cs

```diff
@@ -67,7 +67,7 @@ private BoxContainer MakeRoundEndSummaryTab(string gamemode, string roundEnd, Ti
             //Duration
             var roundTimeLabel = new RichTextLabel();
             roundTimeLabel.SetMarkup(Loc.GetString("round-end-summary-window-duration-label",
-                                                   ("hours", roundDuration.Hours),
+                                                   ("hours", roundDuration.Hours + roundDuration.Days * 24),
```

Comment:
can't you just use TotalHours instead of Hours for the same result and it will continue working for weeks, months, etc

Reply:
I doubt it'll ever go that long, unless you got some _really_ dedicated people

Reply:
yes but it's less characters + less hacky + future proof